cms/static/js/spec/views/assets_spec.js

```diff
@@ -184,7 +184,7 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI", "js/views/asset
                 });
 
                 it('uploads file properly', function () {
-                    var requests = setup.call(this);
+                    var requests;
```

Comment:
I don't understand the purpose of the variable as it isn't used. Can you just delete this line?